ui/src/assets/common.scss

```diff
@@ -657,10 +663,11 @@ button.query-ctrl {
 }
 
 .track-group-panel {
-  --collapsed-background: hsla(190, 49%, 97%, 1);
+  --collapsed-background: var(--main-background-color);
   --collapsed-transparent: hsla(190, 49%, 97%, 0);
```

Comment:
It seems odd that the transparent and expanded variants are not derived from or otherwise tied now to the "main background color" or the collapsed background. The SASS framework provides colour transformation functions that could be employed here.